src/lib/features/environments/environment-oss.e2e.test.ts

```diff
@@ -56,3 +56,14 @@ test('querying environments in OSS only returns environments that are included i
             expect(names).toEqual(['default', 'development', 'production']);
         });
 });
+
+test('querying project environments in OSS only returns environments that are included in oss', async () => {
```

Comment:
we didn't have the test for project envs that would reveal the problem so I added one